apps/remote_control/lib/lexical/remote_control/commands/rename.ex

```diff
@@ -0,0 +1,96 @@
+defmodule Lexical.RemoteControl.Commands.Rename do
+  @moduledoc """
+  We are unable to accurately determine when the rename process finishes,
+  because after the rename, we will receive a series of events like 
+  `DidChange`, `DidSave`, etc., which will trigger expensive actions like compiling the entire project.
+  Therefore, we need this module to tell us if lexical is currently in the process of renaming.
+  """
```

Comment:
Small stylistic thing, but I think this should be a comment instead of a moduledoc.

```suggestion
  # We are unable to accurately determine when the rename process finishes,
  # because after the rename, we will receive a series of events like 
  # `DidChange`, `DidSave`, etc., which will trigger expensive actions like compiling the entire project.
  # Therefore, we need this module to tell us if lexical is currently in the process of renaming.
  
```